crds/tests/test_certify.py

```diff
@@ -719,8 +719,8 @@ def certify_roman_valid_asdf():
     """
     Required Roman test: confirm that a valid asdf file is recognized as such.
     >>> old_state = test_config.setup(url="https://roman-crds-serverless.stsci.edu", observatory="roman", cache=test_config.CRDS_TESTING_CACHE)
-    >>> certify.certify_file("data/roman_wfi16_f158_flat_small.asdf", "roman_0003.pmap", observatory="roman")
-    CRDS - INFO -  Certifying 'data/roman_wfi16_f158_flat_small.asdf' as 'ASDF' relative to context 'roman_0003.pmap'
+    >>> certify.certify_file("data/roman_wfi16_f158_flat_small.asdf", "roman_0007.pmap", observatory="roman")
```

Comment:
Why did this change?

Reply:
i already reverted it back, I just wanted to see the impact to the test results.